### PR TITLE
Fix - Missing get_language_package_uri locale param

### PR DESCRIPTION
**Pull request by shivapoudel**
Fix - Missing get_language_package_uri locale param and tweaks docblocks :)

### CODE REVIEW

**Comment by mikejolley** [comment]
@claudiosmweb is it missing?

File: includes/class-wc-language-pack-upgrader.php [hunk 1]
```diff
@@ -109,21 +109,19 @@ public function has_available_update( $locale = null ) {
...
-	 * Check if language pack exists
-	 *
+	 * Check if language pack exists.
+	 * @param  string $locale
 	 * @return bool
 	 */
-	public function check_if_language_pack_exists() {
-		$response = wp_safe_remote_get( $this->get_language_package_uri(), array( 'timeout' => 60 ) );
+	public function check_if_language_pack_exists( $locale ) {
+		$response = wp_safe_remote_get( $this->get_language_package_uri( $locale ), array( 'timeout' => 60 ) );
```
Comment (shivapoudel) [inline includes/class-wc-language-pack-upgrader.php]: @mikejolley and @claudiosmweb Its missing here :P

**Comment by mikejolley** [comment]
Ah ok. @claudiosmweb maybe for 2.3 too? Everything else is ready pending testing.

**Comment by claudiosmweb** [comment]
Has no missing anything because picks up again with `get_locale()`

**Pull request closed without merging**